docpipeline/generate.py

```diff
@@ -43,6 +43,7 @@
     ],
     "globalMetadata": {{
       "_appTitle": "{package}",
+      "_version": "{version}",
```

Comment:
nit: Adding this under `globalMetadata` sounds good, but to possibly disambiguate what kind of version this is for, consider `_packageVersion` since this will be applied for all pages.

Reply:
Done, thanks.